src/projects/TodoList/index.js

```diff
@@ -69,7 +97,13 @@ class TodoList extends React.Component {
             <p className='pr-1'><strong>Add a new item:</strong></p>
```

Comment:
Quick note on these quote marks, when doing JSX (html) elements, can you use " marks instead of ' marks? Its the standard for writing HTML so it carries across to here, and as you can see Github doesn't like it 😂

Reply:
ahh yeah that looks like asss my bad